src/cmd-build

```diff
@@ -25,11 +25,12 @@ EOF
 
```

Comment:
Let's add it to the usage in `print_help()` too?

Reply:
Yes. I missed that. Will add a usage description.